.github/workflows/keepalive.yml

```diff
@@ -1,10 +1,10 @@
 name: Keepalive
 
 on:
-  workflow_dispatch:
-  # The 8th of every month at 4:50pm UTC
+  # At 8:25 PM UTC, on day 22 of the month
   schedule:
-    - cron: '50 16 8 * *'
+    - cron: '25 20 22 * *'
+  workflow_dispatch:
```

Comment:
Why is this change being done? Seems like a bug to be randomly changing the cron. Is it gonna change the cron every time we run this thing?

Reply:
No it's not going change the cron - it's created using a "predictably random" method - https://github.com/silverstripe/module-standardiser/pull/2/files#diff-e75f39a6d898b74523b527bacefe6bb8245175e773119453e2a8d6f98c881f77R215